examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -28,50 +28,62 @@ using JLD2
 using Oceananigans
 using Oceananigans.Units: minute, minutes, hour
 
-# ## The grid
+# ## A vertically-stretched grid
 #
-# We use 32³ grid points with 2 m grid spacing in the horizontal and
+# We use 32³ grid points with 2 meter grid spacing in the horizontal and
 # varying spacing in the vertical, with higher resolution closer to the
-# surface,
+# surface. We use a two-parameter generating function to specify the
+# vertical cell interfaces:
 
-σ = 1.1 # stretching factor
-Nz = 24
-Lz = 32
+Nz = 24 # vertical resolution
```

Comment:
Nz = 24 # number of points in the vertical direction

Reply:
Thanks --- made that change!

For the future, you can propose specific changes like this by pressing the "+/-" button (left most formatting button). That feature is describe in point 6 here: 

https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request